stack.yaml

```diff
@@ -18,6 +18,16 @@ extra-deps:
 - quickcheck-state-machine-0.6.0
 - command-0.1.1
 
+# persistent-sqlite with CASCADE DELETE support
+#
+# FIXME: host under input-output-hk origanization.
+- git: https://github.com/KtorZ/persistent
+  commit: 79f2ece07eafae005a703c8eda1bd2420b5e07b5
+  subdirs:
+      - persistent-sqlite
```

Comment:
I originally wanted to fork `persistent` on `input-output-hk` but there's already a fork there that is archived.
I've asked for permission to unarchive it and get push rights on it.

Note that, in the future, we might want to do something less hacky than my hack above. There's some work to follow on the matter here:

https://github.com/yesodweb/persistent/pull/912